__tests__/test-data.test.js

```diff
@@ -77,5 +77,51 @@ describe("GET /api", () => {
         }
       });
   });
+});
 
+describe("GET /api/articles", () => {
+  test("200: responds with status code 200", () => {
+    return request(app).get("/api/articles").expect(200);
+  });
+  test("returns an array of correct length filled with article objects with correct properties to the client", () => {
+    return request(app)
+      .get("/api/articles")
+      .then(({ body }) => {
+        const articles = body.articles;
+        expect(articles.length).toBe(13);
+        articles.forEach((item) => {
+          expect(typeof item.author).toBe("string");
+          expect(typeof item.title).toBe("string");
+          expect(typeof item.article_id).toBe("number");
+          expect(typeof item.topic).toBe("string");
+          expect(typeof item.created_at).toBe("string");
+          expect(typeof item.votes).toBe("number");
+          expect(typeof item.article_img_url).toBe("string");
+          expect(typeof item.comment_count).toBe("number");
+        });
+      });
+  });
+  test("objects are sorted by date in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .then(({ body }) => {
+        const articles = body.articles;
```

Comment:
This solution is valid, we generally recommend using jest-sorted for this but it's good to see you found another way of doing things